src/react-native/20-building/building.md

```diff
@@ -10,17 +10,40 @@
 
 In this section, you will:
 
-- TODO
+- Cover EAS production builds
+- Learn how to create a build
```

Comment:
We’ve mostly tried to avoid words like “learn how” and told learners what they will actually do, e.g. “Create a build.”

Also, these should always end in a punctuation mark (period).

Reply:
updated